### ISSUE TITLE
[serial] Com port redirection - EventChar 0x7E ('~') cannot be set 

### ISSUE THREAD

**Issue by T3N4K4** [FreeRDP/FreeRDP#4107]
I am currently trying to redirect a Signotec Signature Pad Gamma ERT (ST-GERT-3-UFT100).

When trying to get the device / communicate with the device via the demo software of signotec, I get the "warning" (and then nothing happens):
```
[17:18:14:726] [21169:21561] [WARN][com.winpr.comm] - EventChar 0x7E ('~') cannot be set
[17:18:14:739] [21169:21587] [WARN][com.winpr.comm] - EventChar 0x7E ('~') cannot be set
[17:18:16:757] [21169:21664] [WARN][com.winpr.comm] - EventChar 0x7E ('~') cannot be set
[17:18:16:770] [21169:21690] [WARN][com.winpr.comm] - EventChar 0x7E ('~') cannot be set 
```

Windows terminal server: 2012R2
Linux 32bit, 15.10.
Latest freerdp-master (f9a47a346904f3e2ecfb228f2e4e5ad113fe6a3c)

When typing "change port" in a cmd in windows I get the result that it was correctly redirected.

Commandline: xfreerdp /u: /p: /d: /v: /serial:COM10,/dev/ttyUSB0 /gdi:hw

The device itself is found on ttyUSB0 (as seen via dmesg).

**Comment by T3N4K4** [comment]
@akallabeth @hardening @nfedera @bmiklautz anyone ? Sorry for tagging, but it is an important matter for me.

**Comment by bmiklautz** [comment]
@T3N4K4 event characters are currently not supported (have a look to winpr/libwinpr/comm/comm_serial_sys.c for details).

**Comment by T3N4K4** [comment]
@bmiklautz Thank you. Should I leave this open until it is implemented or should I close this ?

**Comment by bmiklautz** [comment]
@T3N4K4 as you like. I can mark it as feature.

**Comment by T3N4K4** [comment]
I guess this is fixed by #4214 then.

**Comment by hardening** [comment]
Sidenote: the Signotec Signature Pad Gamma requires some other features to work correctly (especially the XOFF_CONTINUE feature)
